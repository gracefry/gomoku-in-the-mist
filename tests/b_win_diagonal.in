who
place A1
place S19
place B2
place R18
place C3
place Q17
place D4
place P16
place E5
