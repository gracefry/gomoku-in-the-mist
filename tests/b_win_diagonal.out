B
Black wins!
A1S19B2R18C3Q17D4P16E5
Thank you for playing!
